icc-x-api/icc-message-x-api.ts

```diff
@@ -744,42 +753,44 @@ export class IccMessageXApi extends iccMessageApi {
             _.flatMap(parsedRecords.records as ET20_80Data[], r => {
               const errors: StructError[] = []
               let refEt20 = r.et20 && r.et20.reference.trim()
-              if (r.et20 && r.et20.errorDetail)
+              if (r.et20 && r.et20.errorDetail) {
```

Comment:
It shouldn't be changed but as business requirement, et80 can't come without et20 so it should be safe